packages/react-code-editor/src/components/CodeEditor/examples/CodeEditorCustomControl.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { CodeEditor, CodeEditorControl } from '@patternfly/react-code-editor';
+import PlayIcon from '@patternfly/react-icons/dist/esm/icons/play-icon';
+
+export const CustomControlCodeEditor: React.FunctionComponent = () => {
+  const [code, setCode] = React.useState('');
+
+  const onChange = code => {
+    setCode(code);
+  };
+
+  const onExecuteCode = code => {
+    // eslint-disable-next-line no-console
+    console.log(code);
+  };
+
+  const customControl = (
+    <CodeEditorControl
+      icon={<PlayIcon />}
+      aria-label="Execute code"
+      toolTipText="Execute code"
+      onClick={onExecuteCode}
+      isVisible={code !== ''}
+    />
+  );
+
+  return (
+    <>
```

Comment:
Small nit - `<>` shouldn't be needed when there is only a single component being returned.